src/image_file.cpp

```diff
@@ -308,6 +321,14 @@ int ImageFile::init_image_file() {
     bool has_error = false;
     auto lowers = conf.lowers();
 
+    if (!conf.traceLayer().empty()) {
+        m_prefetcher = FileSystem::new_prefetcher(conf.traceLayer());
+        if (m_prefetcher->get_mode() == FileSystem::Prefetcher::Mode::Replay && !lowers.empty()) {
+            LOG_INFO("Prefetch: replay mode detected, will ignore trace layer");
+            lowers.pop_back();
```

Comment:
as the trace layer is passed in by tracelayer in config file,  the trace layer has no need to store in both lowers and tracelayer.